nodes/viz/texture_viewer.py

```diff
@@ -236,7 +257,21 @@ def draw_buttons_ext(self, context, layout):
         layout.separator()
         layout.prop(self, "bitmap_format", text='format')
         layout.separator()
-
+        row = layout.row()
+        row.prop(self, 'color_mode_save', expand=True)
+        layout.separator()
+        if img_format == 'PNG':
+            row = layout.row()
+            row.prop(self, 'compression_level', text='set compression')
+            layout.separator()
+        if img_format == 'JPEG':
```

Comment:
`if img_format in {'JPEG','JPEG2000'}:`      not sure I understand the question.